foundryup/foundryup

```diff
@@ -421,7 +421,7 @@ banner() {
 .xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx
 
 Repo       : https://github.com/paritytech/foundry-polkadot
-Book       : https://book.getfoundry.sh/
+Book       : https://paritytech.github.io/foundry-book-polkadot/
```

Comment:
I think that we need to request a domain for foundry like `foundry-book.polkado.io` or similar

Reply:
you can take this as a template: https://github.com/paritytech/devops/issues/3580